.github/workflows/ci.yml

```diff
@@ -103,6 +109,13 @@ jobs:
           cd build
           cpack .
 
+      - name: Package (Qt)
+        if: ${{ !!matrix.msvc && matrix.config }}
+        run: |
+          cd build
+          windeployqt config.exe --no-compiler-runtime --no-opengl-sw --no-system-d3d-compiler --no-translations --no-quick-import
+        shell: cmd
+
```

Comment:
This is not used by CPack. You need to install runtime dependencies in CMake itself.